setup.py

```diff
@@ -15,27 +15,20 @@
 history = open("CHANGES.rst").read()
 
 tests_require = [
-    "check-manifest>=0.25",
-    "coverage>=4.0",
-    "isort>=4.3.3",
-    "pydocstyle>=2.0.0",
-    "pytest-cov>=2.5.1",
-    "pytest-pep8>=1.0.6",
+    "pytest-invenio>=1.3.2",
```

Comment:
Shel: I believe `flask-resources` should actually be a generic library. Let's use `pytest-flask~=1.0.0` here instead. This will save us elasticsearch being initialized in the test `app` fixture too. 

Marking it as shelved, because I've been using pytest-flask when exploring this code and I will be able to push it and the changes.

Reply:
Totally agree on this one, nice catch :)